### ISSUE TITLE
Issues with RDNS

### ISSUE THREAD

**Issue by gummiboll** [linode/python-linode-api#45]
Hello!

Not sure if I am doing something wrong or if this is a actual bug but I can't seem to set RDNS with this library.

When I try it for IPv4 I get a stacktrace:

```
>>> l.ips.ipv4.public[0].rdns = 'my-rdns'
>>> l.ips.ipv4.public[0].save()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/private/tmp/kaka/lib/python2.7/site-packages/linode/objects/base.py", line 144, in save
    data=self._serialize())
  File "/private/tmp/kaka/lib/python2.7/site-packages/linode/linode_client.py", line 430, in put
    return self._api_call(*args, method=requests.put, **kwargs)
  File "/private/tmp/kaka/lib/python2.7/site-packages/linode/linode_client.py", line 375, in _api_call
    endpoint = endpoint.format(**{ k: str(vars(model)[k]) for k in vars(model) if 'id' in k })
KeyError: 'address'
>>> print l.ips.ipv4.public[0].address
my.ip.addr
```

I also tried sending save() to the linode itself (i.e: `l.save()`), I don't get a stacktrace but the RDNS isn't updated.

For IPv6 I don't seem to see a updatable property for RDNS or a save function.

**Comment by Dorthu** [comment]
Thanks, I'll take a look

**Comment by Dorthu** [comment]
This fix has been released in v4.1.3b1
